tests/python/gpu/test_operator_gpu.py

```diff
@@ -1918,6 +1918,65 @@ def test_softmax_activation():
         assert_almost_equal(cpu_a.grad.asnumpy(), gpu_a.grad.asnumpy(),
                 atol = 1e-3, rtol = 1e-3)
 
+
+@with_seed()
+def test_bilinear_sampler_versions():
+    data = mx.sym.Variable('data')
+    grid = mx.sym.Variable('grid')
+    sym1 = mx.sym.BilinearSampler(data=data, grid=grid)
+    sym2 = mx.sym.BilinearSampler(data=data, grid=grid, cudnn_off=True)
+    sym3 = mx.sym.BilinearSampler(data=data, grid=grid)
+
+    test_cases = [[(1,3,15,16),(1,2,10,10)],
```

Comment:
You could use [parameterized](https://pypi.org/project/parameterized/)

Reply:
Seems like this introduces a new dependency, and many other tests would also need refactoring to migrate to this, so you want me to do this for this test only?

Reply:
Well is was just a suggestion, maybe we could consider it for a future improvement

Reply:
Sure, thanks for the suggestion, I'm supportive of making those changes all together in a separate PR.